src/main/java/com/laytonsmith/core/events/drivers/InventoryEvents.java

```diff
@@ -987,4 +989,90 @@ public Version since() {
 			return MSVersion.V3_3_1;
 		}
 	}
+
+	@api
+	public static class item_pre_anvil extends AbstractEvent {
+
+		@Override
+		public String getName() {
+			return "item_pre_anvil";
+		}
+
+		@Override
+		public String docs() {
+			return "{}"
+					+ " Fires when a recipe is formed in an anvil, but the result has not yet been clicked."
+					+ " { viewers: all human entities looking at this anvil "
+					+ " | first_item: the first item in the anvil"
+					+ "	| second_item: the second item in the anvil"
+					+ " | result: the result of the anvil"
```

Comment:
I would word this as:
`the first ingredient in the anvil recipy`
`the second ingredient in the anvil recipy`
`the result of the anvil recipy`
Or something alike. Again a bit more explicit, leaving less room for misinterpretation (what is the result of an anvil?).

Reply:
Verbage updated.